hack/build-manifests.sh

```diff
@@ -235,8 +235,8 @@ EOM
 
 # Write HCO CRDs
 (cd ${PROJECT_ROOT}/tools/csv-merger/ && go build)
-hco_crds=${TEMPDIR}/hco.crds.yaml
-(cd ${PROJECT_ROOT} && ${PROJECT_ROOT}/tools/csv-merger/csv-merger  --api-sources=${PROJECT_ROOT}/pkg/apis/... --output-mode=CRDs > $hco_crds)
+hco_crds=${PROJECT_ROOT}/config/crd/bases/hco.kubevirt.io_hyperconvergeds.yaml
```

Comment:
This was changed to keep the new layout config folder updated with the new generated HCO CRDs